.circleci/config.yml

```diff
@@ -13,6 +13,8 @@ jobs:
       - run: go mod tidy
       - run: go fmt ./...
       - run: go vet ./...
+      - run: go get -u golang.org/x/lint/golint
```

Comment:
Can you put that command at the beginning, before the checkout ? 
so we separate node installation and check commands

Reply:
Done.